includes/class-wc-calypso-bridge-admin-setup-checklist.php

```diff
@@ -101,13 +116,294 @@ public function admin_menu_structure() {
+				'condition' => false, // TODO Condition logic here. Based on click?
+			),
+
+			array(
+				'title' => __( 'Review shipping', 'wc-calypso-bridge' ),
+				'completed_title' => __( 'View Settings', 'wc-calypso-bridge' ),
+				'description' => __( "We've set up a few shipping options based on your store location. Check them out to see if they're right for you.", 'wc-calypso-bridge' ),
+				'estimate' => '2',
+				'link' => 'admin.php?page=wc-settings&tab=shipping',
+				'condition' => false, // TODO Condition logic here. Based on click?
+			),
+
+			array(
+				'title' => __( 'Add live rates with UPS', 'wc-calypso-bridge' ),
+				'completed_title' => __( 'View Settings', 'wc-calypso-bridge' ),
+				'description' => __( "Showing shipping rates directly from UPS during checkout ensures you're charging customers the right amount for shipping.", 'wc-calypso-bridge' ),
+				'estimate' => '2',
+				'link' => 'admin.php?page=wc-settings&tab=shipping&section=ups',
+				'condition' => ! empty( $ups_settings['user_id'] ) &&
+							   ! empty( $ups_settings['password'] ) &&
+							   ! empty( $ups_settings['access_key'] ) &&
+							   ! empty( $ups_settings['shipper_number'] ),
+				'extension' => 'woocommerce-shipping-ups/woocommerce-shipping-ups.php',
+			),
+
+			array(
+				'title' => __( 'Setup payments with Square', 'wc-calypso-bridge' ),
+				'completed_title' => __( 'View Settings', 'wc-calypso-bridge' ),
+				'description' => __( 'Connect your Square account to accept credit and debit card, to track sales and sync inventory.', 'wc-calypso-bridge' ),
+				'estimate' => '2',
+				'link' => 'admin.php?page=wc-settings&tab=integration&section=squareconnect',
+				'learn_more' => 'https://woocommerce.com/products/square/',
+				'condition' => ! empty( $square_merchant_access_token ),
+				'extension' => 'woocommerce-square/woocommerce-square.php',
+			),
+
+			array(
+				'title' => __( 'Setup payments with PayPal', 'wc-calypso-bridge' ),
+				'completed_title' => __( 'View Settings', 'wc-calypso-bridge' ),
+				'description' => __( 'Connect your PayPal account to let customers to conveniently checkout directly with PayPal.', 'wc-calypso-bridge' ),
+				'estimate' => '2',
+				'link' => 'admin.php?page=wc-settings&tab=checkout&section=ppec_paypal',
+				'learn_more' => 'https://woocommerce.com/products/woocommerce-gateway-paypal-checkout/',
+				'condition' => false, // TODO Condition logic here.
+				'extension' => 'woocommerce-gateway-paypal-express-checkout/woocommerce-gateway-paypal-express-checkout.php',
+			),
+
+			array(
+				'title' => __( 'Setup payments with Stripe', 'wc-calypso-bridge' ),
+				'completed_title' => __( 'View Settings', 'wc-calypso-bridge' ),
+				'description' => __( 'Connect your Stripe account to accept credit and debit card payments.', 'wc-calypso-bridge' ),
+				'estimate' => '2',
+				'link' => 'admin.php?page=wc-settings&tab=checkout&section=ppec_paypal',
```

Comment:
I think this `link` value is wrong - I _think_ it should be `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe`